generated/datafactory/microsoft.datafactory/2017-09-01-preview/types.md

```diff
@@ -74,35 +74,48 @@
 * **dependsOn**: [ActivityDependency](#activitydependency)[]: Activity depends on condition.
 * **description**: string: Activity description.
 * **name**: string (Required): Activity name.
+
 ### AzureMLBatchExecutionActivity
 #### Properties
+* **linkedServiceName**: [LinkedServiceReference](#linkedservicereference): Linked service reference.
+* **policy**: [ActivityPolicy](#activitypolicy): Activity policy.
```

Comment:
Like the change to `appplatform` types above, these properties were previously excluded because the branches are not direct descendants of the enum base shape, and the properties added are defined on the intermediate shape.